core/templates/css/oppia.css

```diff
@@ -1881,6 +1881,22 @@ pre.oppia-pre-wrapped-text {
 .popover-content {
   padding: 10px;
 }
+
+.popover-content:after {
+  background-color: rgb(255,255,255);
+  border-bottom: 1px solid rgba(0,0,0,0.2);
+  border-left: 1px solid rgba(0,0,0,0.2);
+  content: '';
+  height: 16px;
+  left: 0%;
+  margin-left: -8px;
+  margin-top: 8px;
+  position: absolute;
```

Comment:
Can you please explain why these changes are needed and how it fixes the issue. Also, have you tested tooltips in other places?